backend/src/scripts/migrations/4.24/addAdminJob.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
@ChristosTsiotsias this should be in 4.26

4.24 is an RC right now so this won't be coming out in 4.24